ci-operator/step-registry/ipi/conf/gcp/network/ipi-conf-gcp-network-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/install-config-patch.yaml"
+
+if [[ -n "${NETWORK}" ]] && [[ -n "${CONTROL_SUBNET}" ]] && [[ -n "${COMPUTE_SUBNET}" ]]; then
+  cat > "${PATCH}" << EOF
+platform:
+  gcp:
+    network: ${NETWORK}
+    controlPlaneSubnet: ${CONTROL_SUBNET}
+    computeSubnet: ${COMPUTE_SUBNET}
+EOF
+elif test -f "${SHARED_DIR}/xpn.json"; then
```

Comment:
Let us make this step only for xpn network case, so suggest to remove the previous `if` and the later `else` condition, rename the file to `ipi/con/xpn-network`

Reply:
Do you mean `ipi/conf/gcp/xpn-network`?

Reply:
ah, yeah.